src/pages/HomePage/MapSide/MapSide.module.scss

```diff
@@ -0,0 +1,75 @@
+@import '/src/theme/breakpoint.scss';
+
+.mapContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 22px;
+}
+
+.mapContainer form {
+  margin-bottom: 20px;
+  display: contents;
+}
+
+.location {
+  height: 53.5px;
+  margin-right: 10px !important;
+}
+
+.searchBy {
+  font-size: 18px !important;
+  font-weight: 500 !important;
+  color: #00cbb8;
+  letter-spacing: 1.25px !important;
+}
+
+.icoLocat {
+  color: var(--turquoise-color);
+  font-size: 35px !important;
+}
+
+@media screen and (min-width: $screen-xs) {
+  .location {
+    width: 75%;
+    margin-right: 0px !important;
+    margin-bottom: 10px !important;
+  }
+}
+
+@media screen and (min-width: $screen-sm) {
+  .mapContainer form {
+    display: unset;
```

Comment:
Might be better to set it to something specific like `static`. The container is also wider than the elements inside making the elements not centred.